spec/lib/farscape/plugins_spec.rb

```diff
@@ -37,6 +37,25 @@ def call(env)
 
 end
 
+describe Farscape::Agent do
```

Comment:
The only new functionality should be captured here.  Otherwise the behavior is expected to be the same.

However, I should add a test for making sure Agent using doesn't change the environment plugins.